torch/csrc/Module.cpp

```diff
@@ -421,31 +421,49 @@ PyObject *THPModule_safeCall(PyObject *_unused, PyObject *args, PyObject *kwargs
   return result;
 }
 
+static std::string parseString(PyObject *obj)
+{
+  if (PyBytes_Check(obj)) {
```

Comment:
https://docs.python.org/2/c-api/string.html

They're aliased to PyString in Python 2